data/strings/strings.txt

```diff
@@ -20350,6 +20350,147 @@
+    nl = Log in op OpenStreetMap om al je kaartbewerkingen automatisch te uploaden. Meer informatie <a href="https://github.com/organicmaps/organicmaps/issues/6144">hier</a>.
+    pl = Zaloguj się do OpenStreetMap, aby automatycznie przesyłać wszystkie zmiany na mapie. Dowiedz się więcej <a href="https://github.com/organicmaps/organicmaps/issues/6144">tutaj</a>.
+    pt = Por favor, faz o login no OpenStreetMap para carregar automaticamente todas as tuas edições de mapas. Sabe mais <a href="https://github.com/organicmaps/organicmaps/issues/6144">aqui</a>.
+    pt-BR = Faça login no OpenStreetMap para carregar automaticamente todas as edições de mapas que você fizer. Saiba mais <a href="https://github.com/organicmaps/organicmaps/issues/6144">aqui</a>.
+    ro = Vă rugăm să vă conectați la OpenStreetMap pentru a încărca automat toate modificările aduse hărții. Aflați mai multe <a href="https://github.com/organicmaps/organicmaps/issues/6144">aici</a>.
+    ru = Пожалуйста войдите в аккаунт OpenStreetMap.org, чтобы публиковать ваши изменения карты. Подробности по <a href="https://github.com/organicmaps/organicmaps/issues/6144">ссылке</a>.
+    sk = Prihláste sa do OpenStreetMap, aby ste mohli automaticky nahrať všetky svoje úpravy mapy. Viac informácií sa dozviete <a href="https://github.com/organicmaps/organicmaps/issues/6144">tade</a>.
+    sv = Logga in på OpenStreetMap för att automatiskt ladda upp alla dina kartredigeringar. Läs mer <a href="https://github.com/organicmaps/organicmaps/issues/6144">här</a>.
+    sw = Tafadhali ingia kwenye OpenStreetMap ili kupakia kiotomatiki hariri zako zote za ramani. Pata maelezo zaidi <a href="https://github.com/organicmaps/organicmaps/issues/6144">hapa</a>.
+    th = กรุณาเข้าสู่ระบบ OpenStreetMap เพื่ออัปโหลดการแก้ไขแผนที่ทั้งหมดของคุณโดยอัตโนมัติ เรียนรู้เพิ่มเติม<a href="https://github.com/organicmaps/organicmaps/issues/6144">ที่นี่</a>
+    tr = Tüm harita düzenlemelerinizi otomatik olarak yüklemek için lütfen OpenStreetMap'e giriş yapın. Daha fazla bilgi <a href="https://github.com/organicmaps/organicmaps/issues/6144">burada</a>.
+    uk = Будь ласка увійдіть в акаунт OpenStreetMap.org, щоб автоматично публікувати зміни до мапи. Подробиці за <a href="https://github.com/organicmaps/organicmaps/issues/6144">посиланням</a>.
+    vi = Vui lòng đăng nhập vào OpenStreetMap để tự động tải lên tất cả các chỉnh sửa bản đồ của bạn. Tìm hiểu thêm <a href="https://github.com/organicmaps/organicmaps/issues/6144">tại đây</a>.
+    zh-Hans = 请登录OpenStreetMap，自动上传您编辑的所有地图。了解更多信息<a href="https://github.com/organicmaps/organicmaps/issues/6144">请点击这里</a>。
+    zh-Hant = 請登入 OpenStreetMap 自動上傳您所有的地圖編輯內容。了解更多<a href="https://github.com/organicmaps/organicmaps/issues/6144">此處</a>。
+
+  [alert_reauth_message_ios]
+    comment = For iOS message 'alert_reauth_message' should be splitted in two parts: message + link. Here's first part
```

Comment:
@kirylkaveryn is it possible to reuse the translation above, with the embedded HTTP URL, without manual strings splitting? NSMutableAttributedString should support clickable links with "Selectable" text field, right?

@strump strings may not split fine in different languages. Is it possible to dynamically find and cut the HTML tag on iOS?

Reply:
I'm not very good at string processing with `NSString`. I created strings `alert_reauth_message_ios` and `alert_reauth_link_text_ios` by splittinh `alert_reauth_message`. Maybe we can do the same from Objective-C code.